bspump/trigger/__init__.py

```diff
@@ -3,11 +3,13 @@
 from .pubsub import PubSubTrigger
 from .runonce import RunOnceTrigger
 from .trigger import Trigger
+from .crontrig import CronTrigger
```

Comment:
You also need to add the library to https://github.com/bitswan-space/BitSwanPump/blob/4c507110d856add080b5dd3388e48689f2197b7f/setup.py#L76